src/components/Menu/Menu.Styled.js

```diff
@@ -0,0 +1,28 @@
+import styled from 'styled-components';
+import { NavLink as Link } from 'react-router-dom';
+
+export const Wrapper = styled.nav`
+  display: flex;
+  align-items: center;
+  height: 30px;
+  justify-content: space-evenly;
+  width: 60%;
+  margin: auto;
+`
+
+export const NavLink = styled(Link)`
+  flex: 0 0 auto;
+  display: inline-block;
+  line-height: 18px;
+  transition: opacity 0.2s, transform 0.2s;
+  cursor: pointer;
+  color: currentColor;
+  &:hover,
+  &:focus {
+    opacity: 0.8;
+  }
+  &.${(props) => props.activeClassName} {
```

Comment:
NavLink could have helped you here, basically the same functionality `Link` but an active class is appended when the route matches. so this will only require an `.active`